src/main/java/com/cedarsoftware/util/io/MetaUtils.java

```diff
@@ -1108,13 +1110,41 @@ public static boolean isLogicalPrimitive(Class<?> c)
                 c.equals(Class.class);
     }
 
+    /**
+     * Populates a map with a mapping of Class -> Set of Strings
+     */
+    public static Map<Class<?>, Set<String>> loadClassToSetOfStrings(String fileName) {
+        Map<String, String> map = loadMapDefinition(fileName);
+
+        Map<Class<?>, Set<String>> builtMap = new HashMap<>();
```

Comment:
Should we care about the order of builtMap classes - in other words, would it be beneficial to have built map be LinkedHashMap order (insertion) or TreeMap() - alphabetically, or random - hashMap()?  Just asking.